style.css

```diff
@@ -1,3 +1,230 @@
+/* GENERIC STYLING */ 
+
 * {
 	box-sizing: border-box;
+	font-family: Helvetica;
+	font-size: 10px;
+}
+
+h1 {
+	font-family: cursive;
+	color: #ebedf0; /*a grey-ish white*/
+	font-size: 7rem;
+	margin: 20px auto;
+}
+
+h2 {
+	font-size: 3rem ;
+	color: #d65a64; /* cute pink colour*/	
+	margin: 0 50px;
+}
+
+h3 {
+	color: #ebedf0;	
+	font-size: 2rem;	
+}
+
+p {
+	color: #ebedf0;
+	font-size: 1.8rem;
+}
+
+a {
+	text-decoration: none;
+	font-weight: bold;
+	color: #d65a64; 
+	font-size: 1.8rem;
+}
+
+.aboutMe_section, .portfolio_section, .contact_section {
+	padding-top: 30px;
+	padding-bottom: 30px;
+}
+
+/* HEADER SECTION*/
+
+header {
+	display: flex;
+	flex-direction: column;
+	background-color: #262323;
+	padding: 20px;	
+}
+
+nav {
+	display: flex;
+	margin: 20px auto;
+	gap: 20px;
+	justify-content: space-evenly;
+}
+
+nav a {
+	font-size: 2.5rem;
+}
+
+
+/* INTRODUCTION/WELCOME SECTION */
+
+.introduction_section {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-end;
+	justify-content: center;
+	column-gap: 20px;
+	width: 100%;
+	height: 900px;
+	background-image: url(images/horse_head_nebula.jpg); /* a place-holder image */ 
+	background-size:cover; /* allows to get the background in the whole section*/
```

Comment:
i would remove these comments for the following reasons:
- in coding we only reserve comments for when they are very necessary
- the image is probably not a placeholder anymore 😅 and even if it is, no need to specify
- no need to explain background-cover, as that is something we can learn by reading documentation

Reply:
I deleted these comments.